### PR TITLE
Fixed build issue and no transition function issue

### PR DESCRIPTION
**Pull request by kenisteward**
**What kind of change does this PR introduce?** (check one with "x")
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
#41 
#40 

**What is the new behavior?**
Properly builds / draws lines.

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by marjan-georgiev** [comment]
Let's revert the changes to the release folder please. Those files are only built when we release.

**Comment by kenisteward** [comment]
@marjan-georgiev okie doke!  I was publishing to a private repository to test the changes.

**Comment by kenisteward** [comment]
@marjan-georgiev Reverted :)

**Comment by kenisteward** [comment]
@marjan-georgiev @isaacplmann any other comments on this?

File: package.json [hunk 1]
```diff
@@ -137,7 +138,7 @@
     "tslint": "^5.8.0",
     "tslint-config-swimlane": "^3.0.4",
     "tslint-loader": "^3.5.3",
-    "typescript": "^2.6.1",
+    "typescript": "^2.4.2",
```
Comment (isaacplmann) [inline package.json]: Should be ~2.4.2 if you want to keep it below 2.5.  ^2.4.2 will download 2.6.2 right now.

**Comment by isaacplmann** [comment]
I haven't had a chance to pull this down and check it out.  I plan to make some to review things on Monday.

File: package.json (hunk 1 above)
Comment (kenisteward) [reply package.json]: thanks.  changing now.

**Comment by marjan-georgiev** [comment]
Thanks! :+1:

**Comment by kenisteward** [comment]
You're welcome! You guys have awesome charting libraries and I'm glad to
contribute :)

**Pull request merged**